subsys/net/lib/lwm2m/Kconfig

```diff
@@ -23,6 +23,21 @@ config LWM2M_DTLS_SUPPORT
 	select NET_SOCKETS_SOCKOPT_TLS
 	select NET_SOCKETS_ENABLE_DTLS
 
+choice
+	prompt "LwM2M Server object version"
+	default LWM2M_SERVER_OBJECT_VERSION_1_0
+# in the future default LWM2M_SERVER_OBJECT_VERSION_1_1 if LWM2M_VERSION_1_1
```

Comment:
That would just a side comment from my side, I don't think we need this in the code :D

Reply:
OK I will remove this no problem